### ISSUE TITLE
Educational order in the profile

### ISSUE THREAD

**Issue by avkovalevs** [tendenci/tendenci#1073]
Hi @jennyq  
Could you please help with educational order in the profile form?
It seems that it is displayed in alphabetical order by the university name.
Is it possible to display it ordered by the year?

**Comment by avkovalevs** [comment]
Is there any way to change sort order?

**Comment by jennyq** [comment]
Hi @avkovalevs,

Do you mean the "Education History" section on the user profile? Yes, the order can be changed. Just add this line of code under the `class Meta` of `class Education` https://github.com/tendenci/tendenci/blob/master/tendenci/apps/educations/models.py#L47:

    ordering = ('-graduation_year',)

Would you like to do a pull request?

**Comment by avkovalevs** [comment]
Hi @jennyq! 
Yes, I mean the "Education history".
Thank you for details.
I would like to create a pull request.

**Comment by avkovalevs** [comment]
@jennyq 
I added this line to models.py like this:
    ordering = ('-graduation_year',)
```
    class Meta:
#         permissions = (("view_education", _("Can view education")),)
        verbose_name = _("Education")
        verbose_name_plural = _("Educations")
```
However, I can't see some changes in the "Education History".

**Comment by avkovalevs** [comment]
![jeanf](https://user-images.githubusercontent.com/22083701/143243032-8ba106b7-d737-4d9d-b2f6-e7719707f2a8.PNG)

**Comment by jennyq** [comment]
Resolved https://github.com/tendenci/tendenci/commit/cc98cde9b7fc5736ee532536ebe606dde2d7c3e4

**Comment by avkovalevs** [comment]
@jennyq HI!
Today I upgraded the Tendenci to 12.5.11, but I can't see the code changes in tendenci/apps/educations/migrations/0001_initial.py and 
tendenci/apps/educations/models.py.
How can I apply it correctly except manual code changes like yours? 
Thanks

**Comment by lashirley21** [comment]
Hi @avkovalevs, I am the SEO Specialist here at Tendenci. Do you mind sharing which website is using Tendenci software? It'd be nice to know. Thank you for your contributions.

**Comment by jennyq** [comment]
Hi @avkovalevs,

The change hasn't been released yet. For now, you can pull directly from the repo like this:

    pip install  --force-reinstall https://github.com/tendenci/tendenci/archive/refs/heads/master.zip
    python manage.py migrate
    python manage.py deploy

Then restart your site.

**Comment by avkovalevs** [comment]
Thank you @jennyq 
This works. 
How can we get the years sorted in increasing order?

**Comment by avkovalevs** [comment]
Hi @lashirley21 . I'm afraid to show the site to girls with SEO knowledge actually. ;-) When you see our site you will said "Oh men... you are boring, you changed only the logo" )))

**Comment by jennyq** [comment]
> Thank you @jennyq This works. How can we get the years sorted in increasing order?

Currently it is in descending order. You cannot change it to ascending  unless we update it in core. I think descending order **is** more suitable for education listing with most recent or advanced degree first.

**Comment by lashirley21** [comment]
@avkovalevs Ha! No worries. Hey, whenever you feel comfortable, maybe you can help juice up our BuiltWith Profile here https://trends.builtwith.com/cms/Tendenci . You can just run  your website URL on their search. 🤟🏽

**Comment by avkovalevs** [comment]
> Currently it is in descending order. You cannot change it to ascending unless we update it in core. I think descending order **is** more suitable for education listing with most recent or advanced degree first.

@jennyq  I agree with you because this is suitable for most of the communities. However, Claude's team is a group of French Alumni North America and they expect to see the first education (in France) on the top of the list in profile. This is very important for them. That's why they want to get a descending order. 
Is it possible to change the code which allow to choose the order (ASC or DESC)  on the site level common for all members in profiles?

**Comment by jennyq** [comment]
Ok, we'll add a site setting to make it adjustable when we get the chance.

**Comment by avkovalevs** [comment]
@jennyq  Thanks!

**Comment by avkovalevs** [comment]
Hi @jennyq! Is there a chance to get this feature in the next month? :-)

**Comment by jennyq** [comment]
Ok, I'll bump it up in my queue.
